definitions/json/CaseEvent/CaseEvent-HMC-nonprod.json

```diff
@@ -13,7 +13,7 @@
     "EventEnablingCondition": "managingOffice !=\"Unassigned\"",
     "CallBackURLAboutToStartEvent": "",
     "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/addAmendClaimantRepresentative/aboutToSubmit",
-    "CallBackURLSubmittedEvent": ""
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/amendRespondentRepSubmitted"
```

Comment:
Looking at your callbacks PR, should this be this?
```suggestion
    "CallBackURLSubmittedEvent": "${ET_COS_URL}/amendClaimantRepSubmitted"
```

Reply:
And also the file it's changed in, it would only work in non-prod and not in prod